js/checklist.js

```diff
@@ -20,7 +25,9 @@
 	 function modify_import_content_page($) {
 		
 		// Holds the content that will be added to the page
-		var html = "<p>If you would like to incorporate content from a previous iSite, please contact the Academic Technology Group at <a href=\"mailto:atg@fas.harvard.edu\">atg@fas.harvard.edu</a>.</p>";
+		var BASE_COURSE_URL = window.location.pathname.replace("/content_migrations", "");
+		var url = BASE_COURSE_URL + "/external_tools/" + MANAGE_COURSE_TOOL_ID;
+		var html = '<p>If you would like to incorporate content from a previous iSite, click <a href="'+url+'">here</a>.</p>';
```

Comment:
Chatted with @Vittorio2015 about this; he suggested including a `title` attribute on the anchor link.